--- a/ihtml/themes/default/password.tpl	2014-04-23 16:57:04.000000000 +0200
+++ b/ihtml/themes/default/password.tpl	2014-04-24 14:59:07.252001275 +0200
@@ -12,7 +12,7 @@
   <style type="text/css">@import url('themes/default/password-style.css');</style>
   <link rel="shortcut icon" href="favicon.ico">
 
-  <script src="include/prototype.js" type="text/javascript"></script>
+  <script src="/prototype.js" type="text/javascript"></script>
   <script src="include/fusiondirectory.js" type="text/javascript"></script>
   <script src="include/pwdStrength.js" type="text/javascript"></script>
 </head>
