--- a/contrib/bin/fusiondirectory-setup	2015-06-13 04:50:38.792465300 +0200
+++ b/contrib/bin/fusiondirectory-setup	2015-06-13 04:53:51.449429700 +0200
@@ -59,12 +59,12 @@
 
 # fd's directory and class.cache file's path declaration
 my %vars = (
- fd_home          => "/var/www/fusiondirectory",
+ fd_home          => "/usr/share/fusiondirectory",
  fd_cache         => "/var/cache/fusiondirectory",
  fd_config_dir    => "/etc/fusiondirectory",
- fd_smarty_dir    => "/usr/share/php/smarty3",
+ fd_smarty_dir    => "/usr/share/php/Smarty3",
  fd_spool_dir     => "/var/spool/fusiondirectory",
- ldap_conf        => "/etc/ldap/ldap.conf",
+ ldap_conf        => "/etc/openldap/ldap.conf",
  config_file      => "fusiondirectory.conf",
  secrets_file     => "fusiondirectory.secrets",
  locale_dir       => "locale",
