--- contrib/bin/fusiondirectory-insert-schema	2014-02-04 11:00:56.000000000 +0100
+++ contrib/bin/fusiondirectory-insert-schema	2014-03-20 15:28:05.727917718 +0100
@@ -32,7 +32,7 @@
 my $search_cmd      = $ldapsearch." -b \"cn=schema,cn=config\" cn={*}";
 my $list_cmd        = $search_cmd."* cn 2>/dev/null";
 my $cnconfig_cmd    = $ldapsearch." -b \"cn=config\" cn=config dn 2>/dev/null | grep dn:";
-my $path            = "/etc/ldap/schema/fusiondirectory/";
+my $path            = "/etc/openldap/schema/fusiondirectory/";
 my $full_cmd        = "";
 my $ldap_utils_path = "/usr/bin/ldapadd";
 my $schema2ldif     = "schema2ldif";
